src/components/course/CoursePageSidebar.vue

```diff
@@ -114,10 +114,14 @@
     >
       <v-btn
         id="send-invite-btn"
+        :disabled="!isInviteTemplateAvailable"
         @click="sendInvite"
       >
         Send Invite
       </v-btn>
+      <div v-if="!isInviteTemplateAvailable" class="ml-12 mt-2 font-weight-light">
+        In order to send an invite, you must create an 'Invitation' email template.
+      </div>
```

Comment:
If template not created then 'Send' button is disabled and user gets message above.